src/extension/background-script/connectors/lawallet.ts

```diff
@@ -127,8 +129,8 @@ export default class LaWallet implements Connector {
 
     return {
       data: {
-        transactions: parsedTransactions.sort(
-          (a, b) => b.settleDate - a.settleDate
+        transactions: parsedTransactions.filter(
```

Comment:
why is this filter needed? it wasn't there before

Reply:
but I guess it's good to have

Reply:
we moved filtering logic at connector level. filtering was present in getTransactions.ts background script. but for nwc we need to bypass thisf filter which require additional checks so we moved it to connector level